packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -211,7 +180,7 @@ class PoseElement extends React.PureComponent<PoseElementInternalProps> {
   componentDidMount() {
     invariant(
       typeof this.ref !== 'undefined',
-      `No DOM ref found. If you're converting an existing component via posed(Component), you must ensure you're passing the hostRef prop to your underlying DOM element.`
```

Comment:
This should validate against ref being a dom element, splitting this into 2 warnings would be great